gyrokinetic/apps/gkyl_gyrokinetic_run.h

```diff
@@ -14,18 +13,43 @@ struct gkyl_gyrokinetic_timings {
   int num_steps; // Maximum number of time-steps to take.
 };
 
+struct gkyl_gyrokinetic_run_verbosity {
+  bool enabled; // Is verbosity enabled? Prints information every time step. Defaults false
+  double frequency; // Print information with given frequency. Defaults to 1.0
+  bool estimate_completion_time; // Estimate completion time based on current progress. Defaults false
+};
+
 enum gkyl_gyrokinetic_run_app_type {
     GKYL_GK_SINGLEB, // Single-block simulation. Default
     GKYL_GK_MULTIB, // Multi-block simulation
 };
 
 struct gkyl_gyrokinetic_run_inp {
+  enum gkyl_gyrokinetic_run_app_type app_type; // Type of gyrokinetic application to run.
+  union{
+    struct gkyl_gyrokinetic_app *app; // Single-block application input struct.
+    struct gkyl_gyrokinetic_multib_app *app_multib; // Multi-block application input.
```

Comment:
shouldn't these be called `app_inp` and `app_multib_inp`?

Reply:
That's not what the comment says. And even if you are right, why does that need to be stored in this input struct?